Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir /helm-go-lib-build && \
     cd helm-go-lib && \
     go build -o libhelm.so -buildmode=c-shared main.go
 
-FROM ghcr.io/graalvm/graalvm-ce:ol8-java11-21.1.0
+FROM us.gcr.io/broad-dsp-gcr-public/base/jre:11-alpine
```

Comment:
If doing this we can also remove https://github.com/DataBiosphere/leonardo/blob/develop/Dockerfile#L18-L23 in the Dockerifle (and also disable the trivy workflow I think)

Reply:
I tried that here: https://github.com/DataBiosphere/leonardo/pull/2194/commits/fbcd94d74c0fb4339aef5156393deea52299d0a4

Reply:
I changed it to `debian` cuz `alpine` is missing some packages